packages/Frontend/src/components/LargeCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Card, Icon, Image } from 'semantic-ui-react';
+import SealedPortion from './assets/SealedPortion.jpg';
+
+
+export default class LargeCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      starFilled: false,
+    };
+  }
+  toggleFav = () => this.setState({ starFilled: !this.state.starFilled })
+  render() {
+    const { starFilled } = this.state;
+    return (
+      <div style={{ position: 'relative', display: 'inline' }}>
+        <Card>
+          <Icon
```

Comment:
This render function is getting really long. Let's refactor this icon out into a subrender method.